kuma/wiki/jinja2/wiki/document.html

```diff
@@ -386,6 +386,16 @@ <h2 class="offscreen">{{ _('Document Tags and Contributors') }}</h2>
 
 {% block js %}
 
+  <!-- if this document was saved, we need to clear out the cached draft in helpfulness -->
+  <script type="text/javascript">
+    {% if request.GET.document_saved %}
+      var document_saved = 1;
+    {% else %}
+      var document_saved = 0;
+    {% endif %}
+  </script>
+  {% javascript 'helpfulness' %}
```

Comment:
The helpfulness widget has been removed from the site. Please do not include this library.